build.gradle

```diff
@@ -106,22 +83,28 @@ sourceSets {
 	}
 }
 
+tasks.forEach {
+	if (it.hasProperty("duplicatesStrategy")) {
+		it.setProperty("duplicatesStrategy", "WARN")
+	}
+}
```

Comment:
I don't think it is a good idea to set warning for all tasks. Probably, we should check which tasks have this warning and deal with each individual task